js/main.js

```diff
@@ -1,2 +1,87 @@
 /* global data */
 /* exported data */
+// SWITCHING PAGES
+var buttonEntry = document.querySelector('.view-button');
+var buttonNew = document.querySelector('.new');
+
+var enterElement = document.getElementById('test1');
+var viewElement = document.getElementById('test2');
+
+buttonEntry.addEventListener('click', viewClick);
+buttonNew.addEventListener('click', entryClick);
+
+function viewClick(event) {
+  window.location.reload(); // Refresh page
+  enterElement.className = 'view hidden';
+  viewElement.className = 'view';
+}
+
+function entryClick(event) {
+  enterElement.className = 'view';
+  viewElement.className = 'view hidden';
+}
+
+// CREATE ENTRY
+var results = document.querySelector('#entry-form');
+var inputs = document.querySelector('#entry-form').elements;
+results.addEventListener('submit', submitValues);
+
+var pic = document.querySelector('#entry-form img');
+var photo = document.querySelector('#entry-form input[name=\'photoUrl\']');
+var picUpdate = document.querySelector('img');
+
+photo.addEventListener('input', addPhoto);
+
+function addPhoto(event) {
+  picUpdate.src = event.target.value;
+}
+
+function submitValues(event) {
+  event.preventDefault();
+  var obj = {};
+  obj.title = inputs[0].value;
+  obj.photoUrl = inputs[1].value;
+  obj.notes = inputs[2].value;
+
+  obj.entryID = data.nextEntryId;
+
+  data.nextEntryId += 1;
+
+  data.entries.unshift(obj);
+
+  pic.src = 'images/placeholder-image-square.jpg';
+  document.querySelector('#entry-form').reset();
+
+  enterElement.className = 'view hidden';
+  viewElement.className = 'view';
+  window.location.reload(); // Refresh page
+}
+
+document.addEventListener('DOMContentLoaded', renderEntries);
+
+// VIEW ENTRY
+function renderEntries() {
```

Comment:
so you want to break this into 2 functions. 
1. The first should be called when the page loads and loops through the entries
2.  The second should be called from within the first function and the second function should take a single entry and then render the DOM tree and return it. 

I would take a look at the DOM Creation exercise for a good example of calling the render function from within the loop to render and append the entries. 

Make sure to reach out if you have questions about this!